daemon/container_linux.go

```diff
@@ -513,13 +513,21 @@ func (container *Container) buildPortMapInfo(n libnetwork.Network, ep libnetwork
 		networkSettings.MacAddress = mac.(net.HardwareAddr).String()
 	}
 
+	networkSettings.Ports = nat.PortMap{}
+
+	if expData, ok := driverInfo[netlabel.ExposedPorts]; ok {
+		for _, tp := range expData.([]types.TransportPort) {
```

Comment:
Yeah, exposed ports could be provided by other drivers other than the default bridge.
Better to assert. Just tested the change, will push them now.
Thanks.